tests/utils.py

```diff
@@ -19,3 +19,20 @@ def plot_matrix(matrix, output_image_path=None, vmin=None, vmax=None, title=None
     else:
         plt.show()
     plt.close(fig)
+
+
+def plot_waveforms(wf1, wf2=None, wf3=None, title="Untitled"):
```

Comment:
I agree that this is useful. Where do you use it ?

Reply:
I used it temporarily in my code to check some things before I removed it again ^^ I might want to use it for development and debugging later